app/hooks/useSettingsConfig.ts

```diff
@@ -183,6 +193,15 @@ const useSettingsConfig = () => {
         group: t("Integrations"),
         icon: ZapierIcon,
       },
+      {
+        name: t("All Integrations"),
```

Comment:
I think I'd make this "Install integration…" with a PlusIcon, so it looks similar to these

<img width="235" alt="image" src="https://github.com/user-attachments/assets/8b08fcbc-b0aa-4797-83cb-baaaf32480b1" />